Core/COObject+RelationshipCache.m

```diff
@@ -61,6 +61,12 @@ - (void) removeCachedOutgoingRelationshipsForCollectionValue: (id)obj
 		}
 		else
 		{
+            if (![obj isKindOfClass: [COObject class]])
```

Comment:
Is this for NSNull or COBrokenPath? Could to be nice to have a comment that gives an example or may be expand the log description.